.changeset/silver-houses-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': minor
+---
+
+Adds persistence to synced `<Tabs>` so that user's choices are reflected throughout the entire website.
```

Comment:
Suggesting a small tweak here:

```suggestion
Adds persistence to synced `<Tabs>` so that a user's choices are reflected across page navigations.
```